server/src/main/java/org/elasticsearch/cluster/metadata/MetaDataCreateIndexService.java

```diff
@@ -376,6 +371,9 @@ public ClusterState execute(ClusterState currentState) throws Exception {
                 // now, put the request settings, so they override templates
                 indexSettingsBuilder.put(request.settings());
                 if (indexSettingsBuilder.get(SETTING_NUMBER_OF_SHARDS) == null) {
+                    deprecationLogger.deprecated("the default number of shards will change from [5] to [1] in 7.0.0; "
```

Comment:
nit: I think deprecation messages usually begin with a capital letter? Looks that way in most every other place I looked at a quick glance.

Reply:
It's a mix. I dislike them for exception and log messages. It especially grates on me to see the capitalization without a period (as we seem prone to do).